community/kernel/src/main/java/org/neo4j/graphdb/impl/notification/NotificationCode.java

```diff
@@ -40,6 +40,12 @@
     LEGACY_PLANNER( SeverityLevel.WARNING,
                     Status.Statement.DeprecationWarning,
                     "Using PLANNER for switching between planners has been deprecated, please use CYPHER planner=[rule,cost] instead"),
+    PLANNER_UNSUPPORTED( SeverityLevel.WARNING,
+                    Status.Statement.PlannerUnsupportedWarning,
+                    "Using COST planner is unsupported for this query, please use RULE planner instead"),
+    RUNTIME_UNSUPPORTED( SeverityLevel.WARNING,
+                    Status.Statement.RuntimeUnsupportedWarning,
+                    "Using compiled runtime is unsupported for this query, please use interpreted runtime instead"),
```

Comment:
Is the recommended syntax capitals for planners and minuscules for runtimes? Otherwise, we should be consistent here and print `RUNTIME` and `INTERPRETED` instead.